src/api/app/views/webui2/webui/staging/projects/show.html.haml

```diff
@@ -6,6 +6,8 @@
       = render(partial: 'webui2/webui/project/tabs', locals: { project: @staging_project })
       .card-body
         %h3= @pagetitle
+        %p.mt-3
+          Project: #{link_to(@pagetitle, project_show_path(@project.name))}
```

Comment:
Isn't this the same as clicking on the **Overview** tab?

Reply:
yes it is, but is it obvious?

Reply:
It has to be right? It's the only way people can reach `Project#show` by clicking.

I guess the reason it was there was because it was not possible to change the *tabs* from inside the engine.